app/frontend/src/components/apiAccess/Step4.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-container>
+    <h2>API Access for Common Services</h2>Contents to go here
+    <br />
+    <br />
+    <br />
+    <br />
+    <div class="text-center">
+      <v-btn class="BC-Gov-PrimaryButton" text @click="finish()">Finish</v-btn>
```

Comment:
IIRC you can do `@click="finish"` without the parentheses and the method will still invoke as it's parameter-less.

Reply:
Oh yeah, you're right.